### PR TITLE
fix(DB/creature_addon): Remove deprecated bytes2 flags.

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:
- Remove deprecated bytes2 flags.
- Remove couple of wrong creature spawns/addons.

## SOURCE:

(for the deleted creatures)
https://www.wowhead.com/wotlk/npc=21723/blackwind-sabercat
https://www.wowhead.com/wotlk/npc=15652/elder-springpaw
https://www.wowhead.com/wotlk/npc=22100/scorpid-bonecrawler
## Tests Performed:

- tested

## How to Test the Changes:

before running sql:
1) go to the 3 GUIDs that are being deleted, confirm they are out of place.
2) Go to random GUIDs and see their SheatheState (bytes2)
```sql
SELECT * FROM `creature_addon` where `bytes2` = XXXXXXXX;
```
Apply SQL
Go to the GUIDs to confirm their SheatheState (bytes2) is correct.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
Why are they being deleted, though? :P

File: data/sql/updates/pending_db_world/rev_1680463160388009300.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- Remove old/deprecated bytes2 flags.
+UPDATE `creature_addon` SET `bytes2` = 1 WHERE `bytes2` IN (4092,4096,4097,256,257,301993985,318771201,285216769);
+UPDATE `creature_addon` SET `bytes2` = 0 WHERE `bytes2` IN (733,141313,234885121,251662337);
+UPDATE `creature_addon` SET `bytes2` = 2 WHERE `bytes2` IN (258,4098);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680463160388009300.sql:4]: 4096 -> 0

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1680463160388009300.sql:4]: 256 -> 0 going from bitmask

**Comment by avarishd** [comment]
the flags or the creatures?

File: data/sql/updates/pending_db_world/rev_1680463160388009300.sql (hunk 1 above)
Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1680463160388009300.sql:4]: I think 733 -> 1
(1, 4, 8, 16, 64, 128, 512)

**Comment by Gultask** [comment]
those 3 creatures

File: data/sql/updates/pending_db_world/rev_1680463160388009300.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+-- Elder Springpaw, Scorpid Bonecrawler & Blackwind Sabercat wrong spawns.
+DELETE FROM `creature_addon` WHERE `guid` IN (75829,77583,77606,77607,82379);
+DELETE FROM `creature` WHERE `guid` IN (75829,77583,82379);
+
+-- Hatecrest Warrior defensive stance, handled by SAI
+UPDATE `creature_addon` SET `auras` = NULL WHERE `guid` = 51476;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680463160388009300.sql:11]:
```suggestion
-- Hatecrest Warrior defensive stance, handled by SAI
UPDATE `creature_addon` SET `auras` = '' WHERE `guid` = 51476;
```

**Comment by avarishd** [comment]
they are plain wrong, go to them.

**Comment by Gultask** [comment]
image
WHY DO YOU WANT TO SEPARATE THEM!?

Oh, and remember to add id1 in those DELETE queries.

File: data/sql/updates/pending_db_world/rev_1680463160388009300.sql (hunk 1 above)
Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1680463160388009300.sql:4]: 141313 is on 4 mobs (1 of which is wrong and gonna get deleted) beasts

File: data/sql/updates/pending_db_world/rev_1680463160388009300.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+-- Incorrect mob spawns (wrong zone/area/sniffed as a pet (with spirit bond))
+-- Blackwind Sabercat
+DELETE FROM `creature` WHERE (`guid` = 75829 AND `id1` = 21723);
+-- Scorpid Bonecrawler
+DELETE FROM `creature` WHERE (`guid` = 77583 AND `id1` = 22100);
+-- Elder Springpaw
+DELETE FROM `creature` WHERE (`guid` = 82379 AND `id1` = 15652);
+-- Scalewing Serpent
+DELETE FROM `creature` WHERE (`guid` = 73192 AND `id1` = 20749);
+ -- Stunetusk Boar
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1680463160388009300.sql:18]:
```suggestion
-- Stunetusk Boar
```

**Comment by avarishd** [comment]
Most are creatures that cannot have weapons lol.

**Comment by Gultask** [comment]
Which is interesting, actually. The vast majority of creatures in my sniffs have SheatheState = 1, even if they cannot have weapons

**Comment by avarishd** [comment]
Allegedly done, will double check them again.

**Review by Gultask — APPROVED** [review]

**Comment by Gultask** [comment]
```
SELECT * FROM `creature_template_addon` WHERE `bytes2` NOT IN (0,1,2);
SELECT * FROM `creature_addon` WHERE `bytes2` NOT IN (0,1,2);
/* Affected rows: 0  Found rows: 0  Warnings: 0  Duration for 2 queries: 0,140 sec. */
```

**Pull request merged**